src/parser/csql_grammar.y

```diff
@@ -4195,6 +4263,20 @@ drop_stmt
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
 		DBG_PRINT}}
+	| DROP SERVER opt_if_exists identifier
+		{{
+                        PT_NODE *node = parser_new_node (this_parser, PT_DROP_SERVER);
+
+			if (node)
+			  {
+                            node->info.serial.if_exists = $3;
```

Comment:
serial ?
it looks confused with "drop_server".

Reply:
There was a mistake in the copy & paste process.
Thanks for finding the bug.